pep-0649.rst

```diff
@@ -619,7 +621,7 @@ the appropriate namespaces:
 * For methods on classes, and for classes, the locals dictionary
   will be the class dictionary.
 * If the annotations refer to free variables, the closure will
-  be the appropriate tuple containing free variables.
+  be the appropriate tuple containing values for free variables.
```

Comment:
I'm not sure of the terminology here.  The closure tuple contains "cells"; the cells contain values. The free variables retrieve / store the value from inside the cells.

Maybe we should cut through the Gordian knot and just literally say "the appropriate tuple containing cell objects"?j  "the closure tuple"?